runtime/src/bank.rs

```diff
@@ -3579,6 +3594,30 @@ impl Bank {
         self.log_epoch_rewards_sysvar("update");
     }
 
+    /// Create the persistent PDA containing the epoch-rewards data
+    fn create_epoch_rewards_partition_list_account(
+        &self,
+        num_partitions: usize,
+        parent_blockhash: Hash,
+    ) {
+        let epoch_rewards_partition_data = EpochRewardsPartitionDataVersion::V0(PartitionData {
+            num_partitions,
+            parent_blockhash,
+            hasher_kind: HasherKind::Sip13,
+        });
+        let address = get_epoch_rewards_partition_data_address(self.epoch());
```

Comment:
hmm. Thinking a bit more about this. It seems that we might have a problem.
We may have a conflict on the address if we have a fork at the epoch boundary. 
If both block B1 and B2 on different forks are computing the epoch rewards, then they will share the same `address` and overwrites each other's data.

Reply:
the forks will be working on independent views of accounts db though, so that shouldn't be a problem

Reply:
Is this any different than any other account at a specific address seeing different activity in two forks?
I certainly hope the accountsDB handles that correctly.

Reply:
yeah. never mind. They will be stored in different storage.